### PR TITLE
auditd package is called audit in the rhel family

### PR DESCRIPTION
**Pull request by iamthemuffinman**

### CODE REVIEW

File: controls/package_spec.rb [hunk 1]
```diff
@@ -80,8 +80,14 @@
...
-    it { should be_installed }
+  if os.redhat?
+    describe package('audit') do
+      it { should be_installed }
+    end
+  else
+    describe package('auditd') do
+      it { should be_installed }
+    end
   end
```
Comment (chris-rock) [inline controls/package_spec.rb]: @iamthemuffinman lets try to make just one describe block:

```
os.redhat? ? audit_pkg = 'audit'  : audit_pkg = 'auditd'
describe package(audit_pkg) do
  it { should be_installed }
end
```

Comment (iamthemuffinman) [inline controls/package_spec.rb]: @chris-rock Nice improvement. I'll make the changes.

**Comment by iamthemuffinman** [comment]
@chris-rock How's that look?

**Comment by chris-rock** [comment]
Thank you for your contribution @iamthemuffinman

**Comment by iamthemuffinman** [comment]
Thanks @chris-rock and @artem-sidorenko!

**Pull request merged**